test/puppetlabs/trapperkeeper/services/metrics/metrics_testutils.clj

```diff
@@ -1,6 +1,7 @@
 (ns puppetlabs.trapperkeeper.services.metrics.metrics-testutils
   (:import (com.codahale.metrics.graphite GraphiteSender))
-  (:require [schema.core :as schema]))
+  (:require [schema.core :as schema]
+            [clojure.set]))
```

Comment:
I think it would be good to do 
```suggestion
            [clojure.set :as set]))
```
and then update the use of `clojure.set/union` to `set/union`. That is the pattern we tend to follow in other repos.

Reply:
There's no hurry, until the next release we can carry a patch in the package without issue. Thanks!